src/advanced/bitcoin-anchoring.md

```diff
@@ -70,65 +69,37 @@ floor(2/3*N) + 1` is used as a supermajority.
     That means, 7 signatures are required to build an anchoring transaction.
     If `N=4`, then `M=3` signatures are required.
 
-After the necessary amount of signatures is published, any participant
-node can create a correct and signed anchoring transaction and broadcast
-it to the Bitcoin blockchain.
+After the necessary amount of signatures is gathered, a synchronization method
+creates a correct and signed anchoring transaction and commits it to the Bitcoin
```

Comment:
I think just "commit" is enough. Finalized Bitcoin transaction are created not by the synchronization method but during the signature transaction execution.